packages/Confirmation/src/Confirmation.styles.js

```diff
@@ -0,0 +1,25 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+export const confirmStyles = css`
+  padding: ${stylers.spacer(2)};
+`;
+
+export const confirmHeaderStyles = css`
+  ${stylers.fontSize()};
```

Comment:
`;` not needed – if the `styler` function generates an entire line, it should include the `;`.  Up on line 6 where it only generates a value, then the `;` _IS_ needed.